### PR TITLE
[Blog] Week 41

### PR DESCRIPTION
**Pull request by dhodovsk**

### CODE REVIEW

File: content/posts/october-2020 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+## Week 41 (October 5th - October 9th)
+
+- [franta] improved the way we handle jobs. Triggering of them is more reliable -
+  they are being triggered in correct situations ([packit-service#837](https://github.com/packit/packit-service/pull/837))
+
+- We've enhanced the stability of packit-service by adding multiple improvements in our testing procedures.
+
+[franta]: https://github.com/lachmanfrantisek`
```
Comment (jpopelka) [inline content/posts/october-2020:14]:
```suggestion
[franta]: https://github.com/lachmanfrantisek
```

File: content/posts/october-2020 [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+date: 2020-10-15
+draft: false
+---
+
+## Week 41 (October 5th - October 9th)
+
+- [franta] improved the way we handle jobs. Triggering of them is more reliable -
+  they are being triggered in correct situations ([packit-service#837](https://github.com/packit/packit-service/pull/837))
+
+- We've enhanced the stability of packit-service by adding multiple improvements in our testing procedures.
```
Comment (jpopelka) [inline content/posts/october-2020:12]: I don't think users care about our testing. Feel free to remove this :-)

File: content/posts/october-2020 [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: "October 2020"
+date: 2020-10-15
+draft: false
+---
+
+## Week 41 (October 5th - October 9th)
+
+- [franta] improved the way we handle jobs. Triggering of them is more reliable -
+  they are being triggered in correct situations ([packit-service#837](https://github.com/packit/packit-service/pull/837))
```
Comment (jpopelka) [inline content/posts/october-2020:10]:
```suggestion
- [franta] made triggering of jobs more reliable and in correct situations ([packit-service#837](https://github.com/packit/packit-service/pull/837))
```

**Review by jpopelka — COMMENTED** [review]
The ` content/posts/october-2020` has no `.md` suffix

File: content/posts/october-2020 (hunk 2 above)
Comment (dhodovsk) [reply content/posts/october-2020:12]: They care about stability, but sure :)

**Review by TomasTomecek — APPROVED** [review]

**Pull request merged**